addons/controllers/antrea/antreaconfig_util.go

```diff
@@ -0,0 +1,95 @@
+package controllers
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/vmware-tanzu/tanzu-framework/addons/pkg/util"
+
+	clusterapiv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
+
+	cniv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/cni/v1alpha1"
+)
+
+// AntreaConfigSpec defines the desired state of AntreaConfig
+type antreaConfigSpec struct {
+	InfraProvider string `yaml:"infraProvider"`
+
```

Comment:
nitpick: why are there blank lines between properties? Is the linter not complaining?

Reply:
Linter was not complaining, but I did remove all the blank lines